memcached-operator/pkg/controller/memcached/memcached_controller.go

```diff
@@ -2,15 +2,13 @@ package memcached
 
 import (
 	"context"
-	"reflect"
-
 	cachev1alpha1 "github.com/operator-framework/operator-sdk-samples/memcached-operator/pkg/apis/cache/v1alpha1"
+	"reflect"
```

Comment:
Nit: This `"reflect"` change should be reverted. `cachev1alpha1` should not be grouped with other standard library imports.

Out convention is to group standard libs, third-party libs, and project libs separately.